app/helpers/dot_helper.rb

```diff
@@ -8,9 +8,15 @@ def render_agents_diagram(agents)
         } rescue false)
       decorate_svg(svg, agents).html_safe
     else
-      tag('img', src: URI('https://chart.googleapis.com/chart').tap { |uri|
-            uri.query = URI.encode_www_form(cht: 'gv', chl: agents_dot(agents))
-          })
+      uriquery=URI.encode_www_form(cht: 'gv', chl: agents_dot(agents))
```

Comment:
I think @faizalzakaria means that our typical style would have spaces around the equal sign, so

```ruby
uri_query = URI...
```